### PR TITLE
Remove invalid installation content because there is no corresponding files

### PR DESCRIPTION
**Pull request by lixiaobing1**
Remove invalid installstion content because there is no corresponding files.

### CODE REVIEW

File: docs/reference/commandline/cli.md [hunk 1]
```diff
@@ -40,9 +40,6 @@ each `docker` command with `sudo`. To avoid having to use `sudo` with the
 `docker` command, your system administrator can create a Unix group called
 `docker` and add users to it.
 
-For more information about installing Docker or `sudo` configuration, refer to
-the [installation](../../installation/index.md) instructions for your operating system.
```
Comment (thaJeztah) [inline docs/reference/commandline/cli.md]: Can you change this to https://docs.docker.com/engine/installation/ instead?

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/cli.md (hunk 1 above)
Comment (lixiaobing10051267) [inline docs/reference/commandline/cli.md]: ok

**Comment by thaJeztah** [comment]
can you squash your commits? LGTM otherwise

**Comment by lixiaobing10051267** [comment]
ok , has been modified ,and squashed.

**Review by thaJeztah — APPROVED** [review]
Perfect, thanks!

LGTM

**Pull request merged**